.circleci/config.yml

```diff
@@ -217,27 +217,15 @@ jobs:
           command: |
             ulimit -n 8096
             mkdir -p mattermost-server/client/plugins
+            cd mattermost-server/build
```

Comment:
Wondering why do we need to cd here? Is it not possible to do `--env-file=./build/dotenv/test.env`?

Reply:
```
- attach_workspace:
          at: ~/mattermost
```
So we are in `~/mattermost`. I could also change it to go to mattermost-server/build before the `mkdir -p mattermost-server/client/plugins` part or I could go and load the env file from `--env-file mattermost-server/build/dotenv/test.env`.  But I decided to go with this, because the test-schema followup steps are similar to this. So we would keep it somewhat consistent ¯\_(ツ)_/¯ 
WDYT?

Reply:
Yes that's true. Sounds good to me.